tests/runfiles/linux.run

```diff
@@ -440,6 +440,10 @@ tags = ['functional', 'compression']
 tests = ['ctime_001_pos' ]
 tags = ['functional', 'ctime']
 
+[tests/functional/dbufs]
+tests = ['dbufstats_001_pos', 'dbufstats_002_pos']
+tags = ['functional', 'dbufs']
```

Comment:
What we want to be building up is a collection of test cases which verify that all aspects of the ARC are functioning as intended.  Since this covers far more than just dbufs can you rename this directory `tests/functional/arc`, this makes it clear where future ARC specific tests should be added.  There's no need to rename the new `dbufstats` test cases since they are testing a specific aspect of the ARC.